mmpose/models/losses/multi_task_loss.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch.nn as nn
+
+from mmpose.registry import MODELS
+
+
+@MODELS.register_module()
+class MultiTaskLoss(nn.Module):
+    """Combine multi-task losses together with different factors.
+
+    Args:
+        loss_cfg_list (list): List of Loss Config
+        factors (list): List of Loss factors
+    """
+
+    def __init__(self, loss_cfg_list: list, factors: list):
+        super().__init__()
+
+        assert len(loss_cfg_list) == len(factors), (
+            'len(loss_cfg_list) must equal to len(factors')
+
+        self.factors = factors
+        self.num_losses = len(factors)
+
+        losses = []
+        for loss_cfg in loss_cfg_list:
+            t_loss = MODELS.build(loss_cfg)
+            losses.append(t_loss)
+        self.losses = nn.ModuleList(losses)
+
+    def forward(self,
+                input_list,
+                target_list,
+                keypoint_weights=None,
+                epoch_facotrs=None):
```

Comment:
'epoch_facotrs` seems a typo.

More importantly, I think this epoch-based loss weight adjustment should be handled by the head instead of the loss since it's an algorithm-specific behavior. As a loss wrapper, it should have a consistent signature with other losses here.